pypsa-to-imac

```diff
@@ -0,0 +1,400 @@
+            var['Installed capacity|Heat|Solid bio and waste|Primary solid biomass'] = var['Installed capacity|Heat|Solid bio and waste']  
+            
+            var['Installed capacity|Heat|Biomass|CCS'] = 0.001*(n.links.efficiency2.filter(like ='solid biomass CHP CC').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='solid biomass CHP CC').filter(like =country)).sum()
+            var['Installed capacity|Heat|Solid bio and waste|CCS'] = var['Installed capacity|Heat|Biomass|CCS']  
+            var['Installed capacity|Heat|Solid bio and waste|Primary solid biomass|CCS'] = var['Installed capacity|Heat|Solid bio and waste|CCS']  
+          
+            #Capacity :Electricity (Resistive heater, heat pump, DAC(?))
+            
+            var['Installed capacity|Heat|Electricity|Direct'] = 0.001*(n.links.efficiency.filter(like ='resistive heater').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='resistive heater').filter(like =country)).sum()
+            var['Installed capacity|Heat|Electricity|Heat pump'] = 0.001*(n.links.efficiency.filter(like ='heat pump').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='heat pump').filter(like =country)).sum()
+            var['Installed capacity|Heat|Electricity']=0.001*(n.links.efficiency3.filter(like ='DAC').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='DAC').filter(like =country)).sum()+\
+            var['Installed capacity|Heat|Electricity|Direct']+var['Installed capacity|Heat|Electricity|Heat pump']
+
+            #Capacity :Natural gas
+
+            var['Installed capacity|Heat|Gases|Fossil|Natural Gas'] = 0.001*(n.links.efficiency2.filter(like ='gas CHP').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='gas CHP').filter(like =country)).sum()            
+            var['Installed capacity|Heat|Gases|Fossil|Natural Gas'] += 0.001* (n.links.efficiency.filter(like ='gas boiler').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='gas boiler').filter(like =country)).sum()
+            
+            var['Installed capacity|Heat|Gases|Fossil|Natural Gas|CCS'] = 0.001*(n.links.efficiency2.filter(like ='gas CHP CC').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='gas CHP CC').filter(like =country)).sum()            
+            
+            #Capacity :Hydrogen
+            
+            var['Installed capacity|Heat|Gases|Hydrogen'] = 0.001* (n.links.efficiency2.filter(like ='H2 Fuel Cell').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='H2 Fuel Cell').filter(like =country)).sum()  
+            var['Installed capacity|Heat|Gases|Hydrogen'] += 0.001* (n.links.efficiency3.filter(like ='Fischer-Tropsch').filter(like =country)*\
+                 n.links.p_nom_opt.filter(like ='Fischer-Tropsch').filter(like =country)).sum()    
+            
+            #Capacity :Power to gas (?)
+            """    
```

Comment:
Acc. to IAM (Installed capacity|Heat|Gases|P2G) and (Installed capacity|Electricity|Gases|P2G) means : "The installed capacity for power/heat generation,being operated using P2G as fuel. It was included because some models consider that most (or all) the remaining gaseous fuels are P2G." This was removed since we don't consider synthetic gas separate from natural gas and therefore can't separate their production.